physics/GFS_phys_time_vary.fv3.F90

```diff
@@ -469,8 +471,8 @@ subroutine GFS_phys_time_vary_init (
 !$OMP          shared(isbarren_table,isice_table,isurban_table)         &
 !$omp          shared(iswater_table,laim_table,sla_table,bexp_table)    &
 !$omp          shared(stc,smc,slc,tg3,snowxy,tsnoxy,snicexy,snliqxy)    &
-!$omp          shared(zsnsoxy,STYPE,SMCMAX_TABLE,SMCWLT_TABLE,zs,dzs)   &
```

Comment:
Are these changes needed?

Reply:
Maybe they're not needed, but it's making the case consistent which is OK with me.